src/app/video-games/detail/components/button/button.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent {
+
+  @Input()
+  public type = 'primary';
+
+  @Output()
+  public onClick = new EventEmitter();
```

Comment:
Maybe called it `onButtonPressed` to be consistent with other events in other places of the code. I also like the more general naming